refactorlib/parse.py

```diff
@@ -46,7 +46,7 @@ def dictnode_to_lxml(tree, node_lookup=None, encoding=None):
             # to 'UTF8' if we use parser.makeelement()
             parser = XMLParser(encoding=encoding)
             parser.set_element_class_lookup(node_lookup)
-            parser.feed(b'<trash></trash>')
+            parser.feed(b'<a/>')
```

Comment:
yep, no idea.  My guess is libxml has some sort of encoding issue and reads past the end and throws a useless error